PresentScreenings/Utilities/ScreeningsPlanner.cs

```diff
@@ -183,6 +193,8 @@ private bool FitOnDemandScreening(string filmFan, OnDemandScreening onDemandScre
                         found = fits(onDemandScreening);
                         stop = found;
                     }
+
+                    // Try to move a day forward.  **SHOULD tryNextDay NOT BE RESET?**
                     if (tryNextDay)
                     {
                         if (_controller.TryMoveForwardOvernight(onDemandScreening))
```

Comment:
This looks bad in a comment, but I know it is because, sadly enough, Microsoft still doesn't keep up with Mac updates, we can't compile C#.

Please make a user story to cover this hint.

Reply:
Done, it's the #295.